### PR TITLE
Ecm 408

### PR DESCRIPTION
**Pull request by gauravchawla73463**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ECM-408

### Change description ###
Claimant full name implementation changed

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ecm/common/model/ccd/types/ClaimantIndType.java [hunk 1]
```diff
@@ -29,13 +29,19 @@
     private String claimantGender;
 
     public String claimantFullNames() {
-        return String.join(" ", notNullOrEmptyAtt(new ArrayList<>(), Arrays.asList(claimantTitle,
-                claimantTitleOther, claimantFirstNames, claimantLastName)));
+        var fullNameList = Arrays.asList(claimantTitle, claimantFirstNames, claimantLastName);
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ecm/common/model/ccd/types/ClaimantIndType.java:32]: Doesn't this belong in the else part of the if statement below?

File: src/main/java/uk/gov/hmcts/ecm/common/model/ccd/types/ClaimantIndType.java [hunk 2]
```diff
@@ -29,13 +29,19 @@
...
+        if (claimantTitle != null && claimantTitle.trim().equals("Other")) {
+            fullNameList = Arrays.asList(claimantTitleOther, claimantFirstNames, claimantLastName);
+        }
+        return String.join(" ", notNullOrEmptyAtt(new ArrayList<>(), fullNameList));
     }
 
     public String claimantFullName() {
-        return String.join(" ", notNullOrEmptyAtt(new ArrayList<>(), Arrays.asList(claimantTitle,
-                claimantTitleOther, getInitials(), claimantLastName)));
+        var fullNameList = Arrays.asList(claimantTitle, getInitials(), claimantLastName);
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ecm/common/model/ccd/types/ClaimantIndType.java:40]: Doesn't this belong in the else part of the if statement below?

**Review by jthmcts — CHANGES_REQUESTED** [review]
Where are the unit tests?

File: src/main/java/uk/gov/hmcts/ecm/common/model/ccd/types/ClaimantIndType.java (hunk 1 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/ecm/common/model/ccd/types/ClaimantIndType.java:32]: No it doesn't :) FullNameList has a default value, it changes only if the claimantTitle is "Other". I don't think adding else statement will change anything in the way the code works atm.

**Review by jthmcts — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**